css/fonts/font.abril-droidsans.css

```diff
@@ -0,0 +1,143 @@
+/*	Font Abril Fatface & Droid Sans
+
+https://www.google.com/fonts/specimen/Abril+Fatface
+https://www.google.com/fonts/specimen/Droid+Sans
+----------------------------------------------------- */
+/*	VARIABLES
+	BASE
+----------------------------------------------------- */
+/* ICON PATH
+================================================== */
+/* TYPEFACE
+================================================== */
+/* COLOR SCHEME
+================================================== */
+/* UI COLOR
+================================================== */
+/* UI
+================================================== */
+/* Animation
+================================================== */
+/* GFX
+================================================== */
+@font-face {
+  font-family: 'Abril Fatface';
+  font-style: normal;
+  font-weight: 400;
+  src: url(https://fonts.gstatic.com/s/abrilfatface/v12/zOL64pLDlL1D99S8g8PtiKchq-dmiA.ttf) format('truetype');
```

Comment:
Importing fonts from Google will raise privacy issues with many users. There's an issue to purge it from H5P where there are still traces left. https://h5ptechnology.atlassian.net/plugins/servlet/mobile?originPath=%2Fbrowse%2FHFP-2759#issue/HFP-2759

Reply:
Note to myself : there is also a call to https://cdn.knightlab.com/libs/timeline3/latest/css/fonts/font.default.css

Reply:
Oh I am an idiot :D

Reply:
Fixed, thanks ! It also allowed me to discover some hidden bugs.